client/terraform_cli.go

```diff
@@ -77,11 +79,28 @@ func NewTerraformCLI(config *TerraformCLIConfig) (*TerraformCLI, error) {
 		log.Printf("[INFO] (client.terraformcli) persiting Terraform logs on disk: %s", logPath)
 	}
 
+	// Expand any relative paths for variable files to absolute paths
+	var varFiles []string
+	for _, vf := range config.VarFiles {
+		if !strings.HasPrefix(vf, "/") {
```

Comment:
Consider using `path.IsAbs()` for this. A dedicated method makes the purpose more obvious.

Reply:
Actually `filepath.IsAbs()` would probably be better. Already imported and better cross platform support.